lib/src/fnc/string.rs

```diff
@@ -1,41 +1,37 @@
-use crate::ctx::Context;
 use crate::err::Error;
 use crate::fnc::util::string;
 use crate::sql::value::Value;
 
-pub fn concat(_: &Context, args: Vec<Value>) -> Result<Value, Error> {
+pub fn concat(args: Vec<Value>) -> Result<Value, Error> {
 	Ok(args.into_iter().map(|x| x.as_string()).collect::<Vec<_>>().concat().into())
 }
 
-pub fn ends_with(_: &Context, args: Vec<Value>) -> Result<Value, Error> {
-	let args: [Value; 2] = args.try_into().unwrap();
-	let [val, chr] = args.map(Value::as_string);
+pub fn ends_with((val, chr): (String, String)) -> Result<Value, Error> {
 	Ok(val.ends_with(&chr).into())
 }
 
-pub fn join(_: &Context, args: Vec<Value>) -> Result<Value, Error> {
+pub fn join(args: Vec<Value>) -> Result<Value, Error> {
 	let mut args = args.into_iter().map(Value::as_string);
-	let chr = args.next().unwrap();
```

Comment:
This is the fixed potential crash I mentioned.

The function was denoted `Args::Any`, except it unwrapped its first argument.